### PR TITLE
[NOD-1214] Added test for 64 incoming connections to single node 

### PR DESCRIPTION
**Pull request by svarogg**

### CODE REVIEW

**Comment by svarogg** [comment]
Rebuild after update in master

File: integration/64_incoming_connections_test.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	"testing"
+	"time"
+
+	"github.com/kaspanet/kaspad/util/locks"
+
+	"github.com/kaspanet/kaspad/wire"
+)
+
+func Test64IncomingConnections(t *testing.T) {
+	const numBullies = 64 // Much more then 64 hosts creates a risk of running out of available file descriptors
```
Comment (someone235) [inline integration/64_incoming_connections_test.go:15]: then->than

File: integration/64_incoming_connections_test.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	params := make([]*harnessParams, numBullies+1)
```
Comment (someone235) [inline integration/64_incoming_connections_test.go:16]: Everywhere else in the code params is singular of dagParams

File: integration/64_incoming_connections_test.go (hunk 1 above)
Comment (someone235) [inline integration/64_incoming_connections_test.go:15]: Please mention that it's leveldb file descriptors

File: integration/64_incoming_connections_test.go [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	blockAddedWG := sync.WaitGroup{}
+	blockAddedWG.Add(numBullies)
+	for _, bully := range bullies {
+		err := bully.rpcClient.NotifyBlocks()
+		if err != nil {
+			t.Fatalf("Error from NotifyBlocks: %+v", err)
+		}
+
+		bully.rpcClient.onBlockAdded = func(header *wire.BlockHeader) {
+			blockAddedWG.Done()
```
Comment (someone235) [inline integration/64_incoming_connections_test.go:44]: Can you check that it's not called twice?

**Pull request merged**